forgerock-openbanking-model/src/main/java/com/forgerock/openbanking/model/SoftwareStatement.java

```diff
@@ -200,4 +200,8 @@ public String getApplicationId() {
     public void setApplicationId(String applicationId) {
         this.applicationId = applicationId;
     }
+
+    public String getOrganisationId() { return organisationId;}
+
+    public void setOrganisationId(String organisationId) {this.organisationId = organisationId;}
```

Comment:
Minor - this one line format isn't used elsewhere in this class.

I would have been tempted to turn it into a Lombok `@Data` object.

Reply:
I copied that one line style to make it similar to the rest of the class? I don't normally code like that!

Reply:
I suspect IntelliJ is collapsing these methods by default, which is something you can change in the settings. If you view the class in github, you'll see the other methods aren't on one line :)